test/unit/cli.test.js

```diff
@@ -9,7 +9,7 @@ const rootDir = resolve(__dirname, '..', 'fixtures/cli')
 const url = route => 'http://localhost:' + port + route
 
 const nuxtBin = resolve(__dirname, '../../packages/cli/bin/nuxt.js')
-const spawnNuxt = (command, opts) => spawn('node', ['-r', 'esm', nuxtBin, command, rootDir], opts)
+const spawnNuxt = (command, opts) => spawn('node', [nuxtBin, command, rootDir], opts)
```

Comment:
We can directly spawn `nuxtBin` when having shebang :)

Reply:
PS: I think there was another related unit test too

Reply:
Thanks, will change that. Better indeed to keep testing as close to real world usage as much as possible :)